{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--e10e036f-4768-47b6-99ba-81b1c5f77edb",
    "created": "2020-05-26T20:29:55.385672Z",
    "modified": "2020-05-26T20:29:55.385672Z",
    "name": "CVE-2018-7886",
    "description": "An issue was discovered in CloudMe 1.11.0. An unauthenticated local attacker that can connect to the \"\"CloudMe Sync\"\" client application listening on 127.0.0.1 port 8888 can send a malicious payload causing a buffer overflow condition. This will result in code execution, as demonstrated by a TCP reverse shell, or a crash. NOTE: this vulnerability exists because of an incomplete fix for CVE-2018-6892."
}